README.md

```diff
@@ -87,7 +87,21 @@ The following changes are now available in production:
 * Some error response status codes corrected from 403 to 400, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023).
 
 ### individuals-expenses-api
-
+The following changes are now available in production:
+* `RULE_TAX_YEAR_NOT_SUPPORTED` description and message have been updated to indicate that there could be a maximum as well as a minimum supported tax year, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023-1).
+
+* Updated Endpoint: `Create And Amend Employment Expenses`
```

Comment:
Could these be inverted, so it's:
```
 * The descriptions of these endpoints are updated in the developer hub documentation:
   - `Create And Amend Employment Expenses`
   - `Ignore Employment Expenses`
   - (etc)
```
Then just have 2 * entries for "Updated endpoint: ..":
```
  * Endpoint: `Create And Amend Employment Expenses` name updated to include "Create"
```
(+ the same for Create And Amend Other Expenses)

Reply:
That does look cleaner. Updated now. Thanks Matt!